ci/tests/puppeteer/cas.js

```diff
@@ -306,15 +319,39 @@ exports.launchWsFedSp = async (spDir, opts = []) => {
     return exec;
 }
 
+exports.stopGretty = async (gradleDir, deleteDir= true) => {
```

Comment:
It might be relevant to create a general purpose function that is able to launch Gradle with a set of parameters. Then you just invoke this function in the appropriate scripts with the right command. It does not make much sense of cas.js to have a "stopGretty" thing specially more or since it's not obvious what gretty actually is by looking at the function body.

Reply:
done